test/unit/connectors/DelegatedAuthorityConnectorSpec.scala

```diff
@@ -91,28 +90,29 @@ class DelegatedAuthorityConnectorSpec extends UnitSpec with Matchers with ScalaF
 
     "retrieve a single third party delegated authority granted by a user to the given application" in new Setup {
 
-      stubFor(get(urlEqualTo(s"/authority/granted-application/$appId")).willReturn(
+      stubFor(get(urlEqualTo(s"/authority/granted-application/${app.id}")).willReturn(
         aResponse().withStatus(200).withBody(
           s"""
              |{
              |  "application" : {
-             |    "id":"$appId",
-             |    "name":"$appName"
+             |    "id":"${app.id}",
+             |    "name":"${app.name}",
+             |    "trusted": ${app.trusted}
              |  },
              |  "scopes": [
              |    {
-             |      "key":"$scopeKey",
-             |      "name":"$scopeName",
-             |      "description":"$scopeDescription"
+             |      "key":"${scope.key}",
+             |      "name":"${scope.name}",
+             |      "description":"${scope.description}"
              |    }
              |  ],
              |  "earliestGrantDate":$earliestGrantDate
               |}""".stripMargin)
       ))
```

Comment:
That is a good candidate for a quick chat within the team I guess as you could even go further and reuse the stubs from component tests.